client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java

```diff
@@ -51,13 +51,13 @@ public class RemoteShuffleMaster implements ShuffleMaster<RemoteShuffleDescripto
           ThreadUtils.createFactoryWithDefaultExceptionHandler(
               "remote-shuffle-master-executor", LOG));
   private final ResultPartitionAdapter resultPartitionDelegation;
-  private final long rssMetaServiceTimestamp;
+  private final long metaServiceTimestamp;
```

Comment:
meta service is the alias of master? @waitinfuture

Reply:
meta service is lifecyclemanager